devtools/running_packages.rst

```diff
@@ -114,6 +113,214 @@ with:
 
     Read more about the :ref:`deploy() <method_deploy>` method.
 
+.. _using_deploy_generator:
+
+Using deploy generator
+----------------------
+
+With the help of the :ref:`deploy generator <deploy_generator>`, it's possible to have all dependencies of the application to be copied into 
```

Comment:
Too complex sentence. Please rephrase: `The deploy generator is used to have all the dependencies of an application copied into a single place. Then all the files can be repackaged into the distribution format of choice.`

Reply:
done